example.js

```diff
@@ -1,6 +1,8 @@
+/* eslint-disable no-console */
+
 import * as Block from 'multiformats/block'
 import * as codec from '@ipld/dag-cbor'
-import { murmur3128 as hasher } from '@multiformats/murmur3'
+import { murmur3128 as hasher } from 'src'
```

Comment:
was this an auto change? the intention was for this to be copy/pastable, and does `'src'` even work? maybe `'./src/index.js'`?

Reply:
Oops, I think that was VSCode being smart when I moved `index.js` into`src`

Reply:
Have fixed it